src/components/molecules/AppReferralForm.tsx

```diff
@@ -41,7 +41,8 @@ export default ({link, showSublink}: ReferralFormProps) => {
             <div className="flex flex-col gap-4">
                 {showSublink &&
                     <a className="text-center underline font-small-link" onClick={()=>{navigate(AppRoute.REGISTER_REFERRAL_SHEET_SEND_DATE)}}>{translate("link")}</a>}
-                <AppNextButton title={translate("next")} disabled={!file} onClick={() => handleOnClick(link)}/>
+                {/* below, onClick should be handleOnClick, but it's been replaced for the demo */}
```

Comment:
Can we add 'TODO' to this comment? because every change that we are making for the demo it should be fixed later on. It is much easier to find all the TODOs because they are easily searchable with any plugin or IDE

Reply:
OK 👍🏻